### DISCUSSION TITLE
Don't see my coins in the Firefly app balance

### DISCUSSION THREAD

**Discussion by GithubRavilS** [iotaledger/firefly#7743] [Help & Support]
Hi everyone. I've been using the old version of the mobile app Firefly 1.0.7 (4).

Firstly I sent some IOTA coins 1 month ago, it was totally ok.

After that, 3 days ago I opened the app and saw an error message 

"couldn't find dns resolution for the node" 
and the second message 
"Deprecation Announcement
This version of Firefly is deprecated. The IOTA Foundation focuses on enabling core protocol functions through its desktop wallet. To maintain your tokens, we recommend either using Firefly for desktop or a 3rd-party wallet, by importing a Stronghold backup or 24-words backup."

I skipped all the messages and sent the iota coins again directly to the same wallet number as it was 1 month ago. But this time the coins didn't come to my firefly wallet and I don't know what to do for the 3d day. Please help me community🙏🙏

**Comment by begonaalvarezd** [comment]
Firefly mobile was deprecated in July 2023, you should use the desktop version https://firefly.iota.org/. The network also got recently upgraded, that is why your mobile app says `couldn't find dns resolution for the node` because that mobile version is not compatible with the new network, and since mobile is deprecated and unmaintained, we will not update it. 

Just download the latest desktop version and restore your profiles there, if you are missing funds just run the balance finder from settings > advanced
